### PR TITLE
Deploy to beta first, then production

### PR DESCRIPTION
**Pull request by stjohnjohnson**
This ensures we test beta environment first and don't accidentally break production

### CODE REVIEW

File: screwdriver.yaml [hunk 1]
```diff
@@ -34,6 +35,24 @@ jobs:
...
+      - wait-docker: DOCKER_TAG=`git describe --abbrev=0 --tags` ./ci/docker-wait.sh
+      - deploy-k8s: K8S_TAG=`git describe --abbrev=0 --tags` ./ci/k8s-deploy.sh
+      - test: echo Put acceptance tests here
+    environment:
+      DOCKER_REPO: screwdrivercd/ui
+      K8S_CONTAINER: screwdriver-ui
+      K8S_IMAGE: screwdrivercd/ui
+      K8S_HOST: api.k8s.screwdriver.cd
+      K8S_DEPLOYMENT: sdui-beta
+      SD_UI: cd.screwdriver.cd
```
Comment (minz1027) [inline screwdriver.yaml]: why we need this `SD_UI`?

Comment (stjohnjohnson) [inline screwdriver.yaml]: So the functional tests (when added) can test against the right UI

Comment (petey) [inline screwdriver.yaml]: shouldn't this be beta.cd.screwdriver.cd?

**Comment by petey** [comment]
👍

**Pull request merged**